packages/core/src/index.ts

```diff
@@ -1,3 +1,3 @@
-export { list, gql, graphQLSchemaExtension, config } from './schema/schema';
+export { list, gql, config } from './schema/schema';
```

Comment:
```suggestion
export { list, config } from './schema/schema';
```

(`gql` was just there for `graphQLSchemaExtension`)